apps/meteor/app/livechat/server/hooks/sendToCRM.ts

```diff
@@ -60,10 +70,10 @@ const sendMessageType = (msgType: string): boolean => {
 	}
 };
 
-const getAdditionalFieldsByType = (type: string, room: OmnichannelRoomWithExtraFields): AdditionalFields => {
+const getAdditionalFieldsByType = (type: CRMActions, room: OmnichannelRoomWithExtraFields): AdditionalFields => {
```

Comment:
Since this is almost impossible to end-to-end test, what about creating a unit test for this specific function?